Automattic/jetpack
# AI Assistant: Introduce mechanism for calling wpcom API directly for OpenAI streaming calls and Demo JS function for testing

Pair me with D110403-code. 
## Proposed changes:
* Demonstrates generating a JWT and connecting to wpcom api directly to fetch ai responses.

### Other information:

## Jetpack product discussion
pe4Cmx-1ey-p2#comment-593

## Does this pull request change what data or activity we track or use?
no

## Testing instructions:
* Apply D110403-code on your sandbox
* Sandbox `public-api.wordpress.com`.
* On a Jetpack site available to use AI
* Visit the editor
* Open the console
* Run `localStorage.debug='*'`
* Run `askJetpack( 'a paragraph on a beautiful day')`
* Expect to see streamed output like this
  <img width="665" alt="image" src="https://github.com/Automattic/jetpack/assets/746152/566673f8-1e1f-4c92-b19f-0458549cb877">
